Podverse/PodcastsTableViewController.swift

```diff
@@ -229,13 +237,17 @@ extension PodcastsTableViewController:UITableViewDelegate, UITableViewDataSource
     }
     
     func tableView(_ tableView: UITableView, editActionsForRowAt indexPath: IndexPath) -> [UITableViewRowAction]? {
-        let podcastToEditOid = self.subscribedPodcastsArray[indexPath.row].objectID
+        
         let podcastToEditFeedUrl = self.subscribedPodcastsArray[indexPath.row].feedUrl
         
         let deleteAction = UITableViewRowAction(style: .default, title: "Delete", handler: {action, indexpath in
             self.subscribedPodcastsArray.remove(at: indexPath.row)
             tableView.deleteRows(at: [indexPath], with: .fade)
-            PVDeleter.deletePodcast(feedUrl: podcastToEditFeedUrl, moc: self.moc)
+            
+            DispatchQueue.global().async {
```

Comment:
Why not dispatching to global and creating a privateMOC inside the deletePodcast method? Tis way you can avoid dispatching every time you call delete

Reply:
Good catch, I'm updating all the delete methods to dispatch to global inside them.